Packages/Tracking/Core/Runtime/Scripts/PostProcessProvider.cs

```diff
@@ -48,7 +48,7 @@ public enum DataUpdateMode { UpdateOnly, FixedUpdateOnly, UpdateAndFixedUpdate }
         [Tooltip("Whether this post-processing provider should process data received from " +
           "Update frames, FixedUpdate frames, or both. Processing both kinds of " +
           "frames is only recommended if your post-process is stateless.")]
-        public DataUpdateMode dataUpdateMode = DataUpdateMode.UpdateOnly;
+        public DataUpdateMode dataUpdateMode = DataUpdateMode.UpdateAndFixedUpdate;
```

Comment:
What's the reason for this change?

Reply:
Mentioned in the changelog - as everything is set up to use physics interactions now, not updating on FixedUpdate causes interaction issues. Better to default to it being on.

Reply:
The change is actually being introduced in #1648, which still hasn't been merged (hence why this MR is draft)

Reply:
Ah OK, do you want to backport that fix to that PR?

Reply:
Not sure what you mean? That MR needs reviewing before this one, which is why this one's draft

Reply:
I think we're saying the same thing....